include/Util/GameObject.hpp

```diff
@@ -38,6 +38,9 @@ class GameObject {
     void SetDrawable(std::unique_ptr<Core::Drawable> drawable) {
         m_Drawable = std::move(drawable);
     }
+    glm::vec2 GetScaledSize() {
+        return m_Drawable->GetOriginalSize() * m_Transform.scale;
+    };
```

Comment:
group getters together above as per the style guide

Reply:
fixed